include/constants/battle.h

```diff
@@ -170,6 +170,7 @@
 #define STATUS3_AQUA_RING               (1 << 28)
 #define STATUS3_LASER_FOCUS             (1 << 29)
 #define STATUS3_POWER_TRICK             (1 << 30)
+#define STATUS3_SKY_DROPPED             (1 << 31) // Target of Sky Drop
 #define STATUS3_SEMI_INVULNERABLE       (STATUS3_UNDERGROUND | STATUS3_ON_AIR | STATUS3_UNDERWATER | STATUS3_PHANTOM_FORCE)
```

Comment:
Because the semi-invulnerable element of being held in Sky Drop is accounted for by `STATUS3_ON_AIR`, not `STATUS3_SKY_DROPPED`. The only advantage I see for adding `STATUS3_SKY_DROPPED` into `STATUS3_SEMI_INVULNERABLE` would be to simplify some of the clear-status statements, like `gStatuses3[i] &= ~(STATUS3_SKY_DROPPED | STATUS3_ON_AIR);` becoming `gStatuses3[i] &= ~(STATUS3_SEMI_INVULNERABLE);` in src/battle_util.c. If you think that's a necessary change to make, I will make it, but I didn't find it necessary as I was writing the code.

Reply:
sounds good.